README.md

```diff
@@ -128,74 +177,39 @@ Generate a `Red Hat Dependency Analytics Report` for your application by perform
 	}
 	```
 
-1. **Red Hat Dependency Analytics Report** 
-	<br >To generate the `Red Hat Dependency Analytics Report` for your application, you can do one of the following:
-	- Right click on a manifest file *(pom.xml/package.json)* in the *Vscode File explorer* or *Vscode File editor* and choose the `Red Hat Dependency Analytics Report` option.
-	- On an open manifest file, click on the *pie icon* ![icon](images/0.2.0/icon.png) located at the upper right corner in the tab container.
-	- On an open manifest file, hoover over a dependency marked by the inline `Component Analysis`, click on *Quick Fix* and choose *Detailed Vulnerability Report* option.
-	![ screencast ](images/0.3.11/quickfix.gif)
-	- Click on `Open detailed vulnerability report` button inside the vulnerability alert popup message.<br >
-	![ screenshot ](images/screenshots/vuln-alert-popup.png)
-	- Click on `Found <NUMBER> vulnerabilities` in the *status bar* bellow.<br >
-	![ screenshot ](images/screenshots/found-vulns-status-bar.png)
-	- Click on `Dependency analysis in progress` in the *status bar* bellow (only visible when Component Analysis is in progress and not yet complete).<br >
-	![ screenshot ](images/screenshots/dependency-analysis-in-progress-status-bar.png)
-
-	**Bonus Feature** The HTML of the `Red Hat Dependency Analytics Report` will be stored temporarily while the `Red Hat Dependency Analytics Report` tab remains open. Once the tab is closed, the corresponding file will be automatically removed. You can define the filename by [adjusting the configuration](#configuration) in the *extensions workspace settings*, under the *Red Hat Dependency Analytics: Red Hat Dependency Analytics Report File Path* field. The default location for this file is `/tmp/redhatDependencyAnalyticsReport.html`.
+- **Red Hat Dependency Analytics report** 
```

Comment:
Because they were duplicated in the original Quick Start section.  We don't want duplication.  As explained in one of my previous comments, we should describe a few ways to access the report, but we don't need to describe every possible way you can get the report.  Most readers will look at the first couple of ways to do something, and gloss over the rest.  The "Features" section should describe the feature and not how to use the feature, that is what the Quick Start is for, or maybe a stand-alone procedure (in a  separate section if needed).  The "Feature" section should provide the user an overview of the feature, think of this as conceptual information, not procedural information.  I removed the screenshot because it's not helpful, and because of accessibility reasons (such as a  screen reader) we should limit the use of screenshots as a general practice.  I understand that sometime a screenshot can help, but most of the time you can do without them.

Reply:
- I agreed regarding the duplication comment and that the "Feature" section should provide the user with an overview of the feature. In that case it is important to decide on one dedicate place where the user can find a detailed "How to" for every feature (not spread across many sections), so that if we mention that feature in some other place in the document we can refer the user to the "how to" for more detailed instructions. Where should that be? 
- I agree that we can omit some of the ways you can access the report, but lets include the 4 most important ways (i have discussed this with my manager) : Right click on a manifest file, click on the *pie icon*, hoover and click on *Quick Fix* and last, click on `Open detailed vulnerability report` button inside the vulnerability alert popup message. 
- "Most readers will look at the first couple of ways to do something, and gloss over the rest" regarding this quote, we should keep in mind that the README serves as a technical document, comprehensively detailing all the capabilities of the application and allowing the user to find any information he needs inside, regardless of whether he is just skimming or looking for some specific information. We can make the intro more eye catching to reel-in the readers interest and prompt him to try the application but we cannot under-represent the application in the general content. 
- As of my knowledge, screenshots and screen-casts are widely used in application documentations and are a critical asset. It is very helpful to the user, demos the ease of use and interactiveness of the application, that can be very eye catching to the user as well. Regarding accessibility, the format is very standard and is a match to the platforms where the application will be featured, mainly marketplace.  
- Lastly, i was just wondering if you happen to have any guidelines or a template certified by redhat that you use as a reference on how the README document should be built? if yes, could you share it with me please?